cmp/grid/GridModel.js

```diff
@@ -55,23 +55,25 @@ export class GridModel {
     // Immutable public properties
     //------------------------
     /** @member {BaseStore} */
-    store = null;
+    store;
     /** @member {StoreSelectionModel} */
-    selModel = null;
+    selModel;
     /** @member {boolean} */
-    treeMode = false;
+    treeMode;
     /** @member {GridStateModel} */
-    stateModel = null;
+    stateModel;
     /** @member {ColChooserModel} */
-    colChooserModel = null;
+    colChooserModel;
     /** @member {function} */
-    rowClassFn = null;
+    rowClassFn;
     /** @member {function} */
-    contextMenuFn = null;
+    contextMenuFn;
+    /** @member {function} */
+    groupSortFn;
     /** @member {boolean} */
-    enableExport = false;
+    enableExport;
     /** @member {object} */
-    exportOptions = null;
+    exportOptions;
```

Comment:
Sorry for the extra churn here but we've been trying to knock these out where applicable.